substrate/frame/revive/src/tests/pvm.rs

```diff
@@ -3685,26 +3685,18 @@ fn immutable_data_works() {
 }
 
 #[test]
-fn sbrk_cannot_be_deployed() {
-	let (code, _) = compile_module("sbrk").unwrap();
+fn sbrk_cannot_be_linked() {
```

Comment:
since polkavm `0.30.0` `sbrk` instruction is not available. 

Thus I am not sure if it makes sense keeping this manual check of `sbrk` instruction:
https://github.com/paritytech/polkadot-sdk/blob/a8fe52467206246caceae73cde9eeace24b4085a/substrate/frame/revive/src/limits.rs#L213-L216

I don't know, for contracts compiled before the change (not sure if there's even such use case) 🤔 

@athei @koute

Reply:
Yes, but I wonder if we should leave this check for programs built with older `polkavm` versions.